packages/element-web/default.nix

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  jq,
+  yarn,
+  fixup_yarn_lock,
+  nodejs,
+  jitsi-meet,
+}: let
+  pinData = import ./pin.nix;
```

Comment:
I wonder why do you need to have this pinData and separate pin.nix, only for the version number? As I see few lines later you remove the hashes part of it.

Maybe it could be just simpler to put `  "version" = "1.11.47";` here and remove the pin.nix

Reply:
pin.nix defines the webSrcHash. 
Used in packages/element-web/default.nix to fetch correct version of sources from git to be patched.

Reply:
No changes here, hash is used from pin.nix file